content/en/docs/ambient/usage/l4-policy/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Use Layer 4 security policy
+description: Supported security features when only using the secure L4 overlay.
+weight: 20
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+The layering of {{< gloss >}}ztunnel{{< /gloss >}} and {{< gloss >}}waypoint{{< /gloss >}} proxies in Istio's ambient mode gives you a choice on whether or not you want to enable Layer 7 (L7) processing for a given workload.
+
+The Layer 4 (L4) features of Istio's [security policies](/docs/concepts/security) are supported by ztunnel, and are available in ambient mode. [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) also continue to work if your cluster has a {{< gloss >}}CNI{{< /gloss >}} plugin that supports them, and can be used to provide defense-in-depth.
+
+To use L7 policies, and Istio's traffic routing features, you can [deploy a waypoint](/docs/ambient/usage/waypoint) for your workloads.
+
+## Layer 4 authorization policies
+
+The ztunnel proxy performs authorization policy enforcement when a workload is enrolled in secure overlay mode (i.e. with no waypoint proxy configured).
+
+The actual enforcement point is at the receiving (server-side) ztunnel proxy in the path of a connection.
+
+A basic L4 authorization policy looks like this:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+ name: allow-sleep-to-httpbin
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+ action: ALLOW
+ rules:
+ - from:
+   - source:
+       principals:
+       - cluster.local/ns/ambient-demo/sa/sleep
+EOF
+{{< /text >}}
+
+The behavior of the `AuthorizationPolicy` API has the same functional behavior in Istio ambient mode as in sidecar mode. When there is no `AuthorizationPolicy` provisioned, then the default action is `ALLOW`. Once a policy is provisioned, pods matching the selector in the policy only allow traffic which is explicitly allowed. In this example, pods with the label `app: httpbin` only allow traffic from sources with an identity principal of `cluster.local/ns/ambient-demo/sa/sleep`. Traffic from all other sources will be denied.
```

Comment:
This is not entirely true.... Authzpolicy is the one that have biggest gotchas.  In this particular example (L4 authz policy), you can say it is same functional behavior.

Reply:
s/`AuthorizationPolicy` API/L4 `AuthorizationPolicy` API/